package exceptionhandile;

public class arrayindexoutofboundexception {

	public static void main(String[] args) {
		
		int n[] = {1,3,5};
		
		
		
		try {
			System.out.println(n[5]);
		}catch(ArrayIndexOutOfBoundsException e) {
			
		}
		
		

	}

}
